include/nbl/builtin/glsl/ext/Scan/scan.glsl

```diff
@@ -0,0 +1,146 @@
+#ifndef _NBL_GLSL_EXT_SCAN_INCLUDED_
+
+#include <nbl/builtin/glsl/workgroup/arithmetic.glsl>
+#include <nbl/builtin/glsl/math/typeless_arithmetic.glsl>
+
+#define STRIDED_IDX(i) (((i) + 1)*(nbl_glsl_ext_Scan_Parameters_t_getStride())-1)
```

Comment:
no point having something as a define that isnt a compile time constant expression

Reply:
put it there for legibility, removing

Reply:
you can make it into a function though